contracts/AirdropCampaignData.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+/// @title AirdropCampaignData - Data contract for storing of daily merkleRootHashes of airbro Campaigns
+contract AirdropCampaignData {
+    address public admin = 0xF4b5bFB92dD4E6D529476bCab28A65bb6B32EFb3;
+    uint16 public batchHashArrayLimit = 600;
```

Comment:
remove this and next one, and remove checks, waste of gas.
Admin cal always simulate tx before sending it + metamask does that automatically

Reply:
Yes. Now that I think about it, the backend can just check the length of the array before submitting, so I agree this check is unnecessary.

Reply:
Checked with team - the variable will be removed.